scripts/Elemental_Analysis.py

```diff
@@ -32,6 +38,11 @@ def table_right_clicked(self, item):
     def table_changed(self, items):
         print("Table Changed: {}".format([i.symbol for i in items]))
 
+    def select_data_file(self):
+        filename = str(QtGui.QFileDialog.getOpenFileName())
+        if filename != "":
+            self.ptable.set_peak_datafile(str(filename))
+
```

Comment:
Seeing as this is currently in `Elemental_Analysis.py`, and will soon become an overall widget - do you think it's worth creating a new unit test file for this now, or having it be manually tested and add this test in later when it becomes a complete widget, rather than a run script?

Reply:
You can make a generic script for now. Then we wont forget to do it later.